### PR TITLE
Update Person class to use PayRate instead of EmploymentType

### PR DESCRIPTION
**Pull request by h4ow3i**
In this commit, the Person class has been refactored to replace the existing EmploymentType field with a new PayRate field. 

This change aligns with the evolving requirements of the application, where pay rates are now being tracked instead of employment types. 

The commit also includes relevant updates to method signatures, data parsing logic, and associated tests to accommodate the transition from employment types to pay rates.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public class AddCommand extends Command {
             + PREFIX_LASTNAME + "LAST NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_SEX + "SEX "
-            + PREFIX_EMPLOYMENTTYPE + "EMPLOYMENT TYPE "
```
Comment (jxunze) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:32]: can add space behind

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -38,7 +38,7 @@ public class AddCommand extends Command {
             + PREFIX_LASTNAME + "Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_SEX + "m "
-            + PREFIX_EMPLOYMENTTYPE + "ft "
```
Comment (jxunze) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:41]: ditto

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 3]
```diff
@@ -59,7 +59,7 @@ public PersonCard(Person person, int displayedIndex) {
         name.setText(person.getName().value);
         phone.setText(person.getPhone().value);
         sex.setText(getFullSexString(person.getSex().value));
-        employmentType.setText(getFullEmploymentTypeString(person.getEmploymentType().value));
```
Comment (jxunze) [inline src/main/java/seedu/address/ui/PersonCard.java:62]: hmm, can change? to .value? follow the standard from getName and getPhone

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 4]
```diff
@@ -59,7 +59,7 @@ public PersonCard(Person person, int displayedIndex) {
...
+        payRate.setText(person.getPayRate().toString());
         address.setText(person.getAddress().value);
         bankDetails.setText(person.getBankDetails().value);
         hoursWorked.setText(person.getWorkHours().toString());
```
Comment (jxunze) [inline src/main/java/seedu/address/ui/PersonCard.java:65]: ditto

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json [hunk 5]
```diff
@@ -4,7 +4,7 @@
       "firstName": "Alice",
       "lastName": "Pauline",
       "sex": "invalidsex",
-      "employmentType": "ft",
```
Comment (jxunze) [inline src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:7]: can standardise the parsing? I think parse as string better.

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json [hunk 6]
```diff
@@ -5,7 +5,7 @@
       "firstName": "Alice",
       "lastName": "Pauline",
       "sex": "f",
-      "employmentType": "ft",
```
Comment (jxunze) [inline src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json:8]: ditto

File: src/test/java/seedu/address/testutil/TypicalPersons.java [hunk 7]
```diff
@@ -35,7 +35,7 @@ public class TypicalPersons {
         .withFirstName("Alice")
         .withLastName("Pauline")
         .withSex("f")
-        .withEmploymentType("ft")
```
Comment (jxunze) [inline src/test/java/seedu/address/testutil/TypicalPersons.java:38]: I think store as string bah? or do u think int better

**Review by jxunze — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/testutil/TypicalPersons.java (hunk 7 above)
Comment (h4ow3i) [reply src/test/java/seedu/address/testutil/TypicalPersons.java:38]: i think standardise payrate as double

File: src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json (hunk 6 above)
Comment (h4ow3i) [reply src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json:8]: fixed

File: src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json (hunk 5 above)
Comment (h4ow3i) [reply src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json:7]: changed to double

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (h4ow3i) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:32]: fixed

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 2 above)
Comment (h4ow3i) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:41]: fixed

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 3 above)
Comment (h4ow3i) [reply src/main/java/seedu/address/ui/PersonCard.java:62]: As mentioned, value of PayRate is a double, hence need to use toString method

**Review by jxunze — APPROVED** [review]

**Pull request merged**